src/views/product-landing/helpers/transform-raw-content-to-prop.ts

```diff
@@ -39,12 +41,31 @@ export async function transformRawContentToProp(
   /**
    * Build ProductLandingBlock[]
    */
+  const INLINE_CONTENT = await getInlineContentMaps(blocks)
   const transformedBlocks: ProductLandingBlock[] = blocks.map(
     (block: ProductLandingContentBlock) => {
       const { type } = block
       switch (type) {
         case 'heading':
           return { ...block, headingSlug: makeHeadingSlug(block.heading) }
+        case 'tutorial_cards':
+          return {
+            type: block.type,
+            tutorialCards: block.tutorialSlugs.map((slug: string) => {
```

Comment:
For raw `tutorial_cards` and `collection_cards` content blocks, we map the author-provided slugs to card data, using the `INLINE_CONTENT` we fetched, as well as the `formatTutorialCard` and `formatCollectionCard` helpers from the respective target components we'll be rendering in the UI.